src/scss/components/_list.scss

```diff
@@ -100,4 +109,99 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      margin-left: 0;
```

Comment:
this approach we used on footer to reset grid. I thinks after our conversation about grid we don't need to reset this.

Reply:
If I don't reset the margin-left we don't get the expected result. 😕